### ISSUE TITLE
The stargazer channel page has some problems as seen here. There is also a lot of blue which does not match the white background.

### ISSUE THREAD

**Issue by ScratchOSTeam** [lightbeam24/CustomTube#91]
![image](https://github.com/lightbeam24/CustomTube/assets/99697031/945714cb-76db-4f6d-ac19-f87d419fc51c)

**Comment by lightbeam24** [comment]
Yes, these channel pages still need some work. But I didn't think the blue was an issue. Do you know what the default channel colors used to be? Looking at archives all the channels are different colors, so I based it off of Jawed's channel, which used this same blue color.

**Comment by zaxel-github** [comment]
i think the default was just a normal white or grey back then.

**Comment by Techfromthepast123** [comment]
I think it'll be cool if for the customtube layouts from 2008-2012 (before channel banners layouts) to change channel pages's colors to whatever we want.

**Comment by bangbangyouall** [comment]
> I think it'll be cool if for the customtube layouts from 2008-2012 (before channel banners layouts) to change channel pages's colors to whatever we want.

maybe the most common color of the banner could be the color (though that COULD be hard.)
